frontend/src/features/idir/reporting/forms/subcomponents/ReportDateTimePickers.tsx

```diff
@@ -2,17 +2,25 @@ import { FormControl, FormLabel } from "@mui/material";
 import { DateTimePicker, LocalizationProvider } from "@mui/x-date-pickers";
 import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
 import dayjs, { Dayjs } from "dayjs";
+import duration from "dayjs/plugin/duration";
 import { useFormContext } from "react-hook-form";
 
 import { PaymentAndRefundSummaryFormData } from "../../types/types";
 import { RequiredOrNull } from "../../../../../common/types/common";
 
+dayjs.extend(duration);
+const THIRTY_DAYS = 30;
```

Comment:
do we need to handle 28 or 31 day months? would the user just be expecting here to select a month limit regardless of day limit?

Reply:
We might have to handle for 31 days so its easier on the user

Reply:
Replaced with 31 days as the max limit